qemu/tests/migration_virtio_mem_ignore_shared.py

```diff
@@ -0,0 +1,99 @@
+import ast
+import time
+
+from virttest import error_context
+from virttest import utils_test
+
+from virttest.utils_misc import normalize_data_size
+
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Migrates a guest with virtio-mem device and x-ignore-shared enabled
+    1) Boot source and destination VMs with a virtio-mem device
+       and shared memory.
+    2) Enable x-ignore-shared capability in both hosts.
+    3) Check the capability is enabled.
+    4) Start stress tool in source VM.
+    5) Migrate VM.
+    6) Memory tool stress is stopped and migration completed.
```

Comment:
Hi @mcasquer 
I have a confirmation about this step.
The stress is stopped before the migration is completed? or the stress is stopped after the migration is completed?

Reply:
@YongxueHong when testing manually, the stress is stopped in the exact moment the migration is completed. Until that moment the tool is running. As part of the migration the process gets hung automatically.

Reply:
> @YongxueHong when testing manually, the stress is stopped in the exact moment the migration is completed. Until that moment the tool is running. As part of the migration the process gets hung automatically.

Hi @mcasquer 
Do you mean that the stress test should be running during the migration?  and it should be kept running even after the migration is completed. right?

Reply:
@YongxueHong yes, that's the idea until the status of the source vm is stopped.

Reply:
> @YongxueHong yes, that's the idea until the status of the source vm is stopped.

Thanks, got it. I think this is a must condition that the stress should keep running until the source VM is stopped.
So for that, we should add the steps to ensure the stressing is running in the background, for example, call the `Stress.app_running` to check it, since we can not know what will happen in the future.

Add the checking step, we can make sure our test case can run correctly.